DigitalOffice/Services/ProjectService/Repositories/GetProjectInfoByIdRepository.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using ProjectService.Database;
+using ProjectService.Database.Entities;
+using ProjectService.Mappers.Interfaces;
+using ProjectService.Models;
```

Comment:
Using directives are not required by the code and can be safely removed

Reply:
fixed